crates/store/re_protos/proto/rerun/v1alpha1/manifest_registry.proto

```diff
@@ -0,0 +1,361 @@
+syntax = "proto3";
+
+package rerun.manifest_registry.v1alpha1;
+
+import "rerun/v1alpha1/common.proto";
+
+service ManifestRegistryService {
+  // register new partitions with the Dataset
+  rpc RegisterPartitions(RegisterPartitionsRequest) returns (RegisterPartitionsResponse) {}
+
+  // unregister partitions from the Dataset
+  rpc UnregisterPartitions(UnregisterPartitionsRequest) returns (UnregisterPartitionsResponse) {}
+
+  // list partitions in the Dataset
+  rpc ListPartitions(ListPartitionsRequest) returns (ListPartitionsResponse) {}
+
+  // create manifests for all partitions in the Dataset
+  rpc CreateManifests(CreateManifestsRequest) returns (CreateManifestsResponse) {}
+
+  // Query Dataset and return a dataframe containing relevant chunk IDs
+  rpc QueryDataset(QueryDatasetRequest) returns (stream QueryDatasetResponse) {}
+
+  // Get all chunks for a single partition
+  rpc GetAllChunks(GetAllChunksRequest) returns (stream GetAllChunksResponse) {}
+
+  // Get partition chunks that contain data for the specified time range
+  rpc GetChunksRange(GetChunksRangeRequest) returns (stream GetChunksRangeResponse) {}
```

Comment:
I would expect this to be `GetChunks` endpoint that simply takes a wrapped `QueryDatasetRequest`, giving us a generalized way of fetching data (by partition ID, by chunk ID, by query, by entity_path... it's just magic).
```proto
rpc GetChunks(GetChunksRequest) returns (stream GetChunksResponse) {}

message GetChunksRequest {
  QueryDatasetRequest request = 1;
}

message GetChunksResponse {
  rerun.common.v1alpha1.RerunChunk chunk = 1;
}
```